modules/custom/d_content/modules/d_content_init/src/Services/ConfigUpdate.php

```diff
@@ -42,13 +42,13 @@ class ConfigUpdate {
    *   Config factory.
    * @param \Drupal\Core\Extension\ModuleHandler $moduleHandler
    *   Module handler.
-   * @param \Drupal\Core\Logger\LoggerChannelFactory $loggerFactory
+   * @param \Drupal\Core\Logger\LoggerChannelFactoryInterface $logger
    *   Logger factory.
    */
-  public function __construct(ConfigFactory $configFactory, ModuleHandler $moduleHandler, LoggerChannelFactory $loggerFactory) {
+  public function __construct(ConfigFactory $configFactory, ModuleHandler $moduleHandler, LoggerChannelFactoryInterface $logger) {
```

Comment:
o rany jakie lenistwo... a pozostałe klasy to co ? juz zmieniajmy wszystko na interfacy w DI :pray: